api/internal/models/subscriber.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+import "time"
+
+type Subscriber struct {
+	ID           int
+	UserID       int
+	SubscriberId int
+	CreatedAt    time.Time
+	CreatedBy    int
+	UpdatedAt    time.Time
+	UpdatedBy    int
+	DeletedAt    time.Time
+	DeletedBy    int
```

Comment:
This field should be reconsidered as its purpose might not be well served for the requirements of the assignment.